custom-fetcher/index.ts

```diff
@@ -8,12 +8,29 @@ export interface FetcherOptions<TQueryParams = never, TBody = never, THeaderPara
 
 const JSON_HEADERS = ['application/json'];
 
+export interface PaginationHeaders {
```

Comment:
`oats` should not be aware of PaginationHeaders, as this is specific to the gitness service